funding/manager.go

```diff
@@ -1232,6 +1236,16 @@ func (f *Manager) handleFundingOpen(peer lnpeer.Peer,
 		return
 	}
 
+	// If there's a maxopenchannels value set, then we compare the number of
+	// currently open channels against the max.
+	if f.cfg.MaxOpenChannels > 0 && len(channels) >= f.cfg.MaxOpenChannels {
```

Comment:
This limits the maximum number of channels a single peer can open, not on the whole node.  Would need to get all open channels that the node rather than use `FetchOpenChannels(peerPubKey)`